ci_cd/tasks/update_deps.py

```diff
@@ -70,10 +72,16 @@ def update_deps(  # pylint: disable=too-many-branches,too-many-locals,too-many-s
         fail_fast: bool = fail_fast  # type: ignore[no-redef]
         pre_commit: bool = pre_commit  # type: ignore[no-redef]
         ignore_separator: str = ignore_separator  # type: ignore[no-redef]
+        verbose: bool = verbose  # type: ignore[no-redef]
 
     if not ignore:
         ignore: list[str] = []  # type: ignore[no-redef]
 
+    if verbose:
+        LOGGER.setLevel(logging.DEBUG)
+        LOGGER.addHandler(logging.StreamHandler(sys.stdout))
+        LOGGER.debug("Verbose logging enabled.")
```

Comment:
Codecov reports that these lines are not covered by tests. Could/should they be tested?

Reply:
No, they are log messages, not part of the actual code as such. As a solution I can either: Run the tests with `--verbose` or add a `# pragma: no cover` to not include this block (these three lines) in the code coverage statistics?

Reply:
If there is a simple way to check that `--verbose` works (i.e. produces additional messages), I would prefer that. If that is complicated or you consider it to be unnecessary, then I accept the the no cover-solution.

Reply:
It's not complicated - I can add an extra separate test for it :)